src/ExtensionsCommon/NativeInstanceMethodsCodeInjector.cpp

```diff
@@ -103,7 +103,10 @@ namespace Instrumentation
 #if defined(_M_IX86)
 		const auto NativeObjThisPointerElementType = ELEMENT_TYPE_I4;
 #endif
-#if (!defined(_M_X64)) && !defined(_M_IX86)
+#if defined(_M_ARM64)
+        const auto NativeObjThisPointerElementType = ELEMENT_TYPE_I8;
```

Comment:
nit: Please line up with rest of code (add more space at beginning).

Reply:
Gah tabs vs spaces